package.json

```diff
@@ -273,7 +273,8 @@
         "darwin"
       ],
       "architectures": [
-        "x86_64"
+        "x86_64",
+        "arm64"
```

Comment:
@arthurlockman @gregg-miskelly do you have some context why `-x64.zip` is installed for `arm64`?

Reply:
In order to debug x64 processes on an M1